tests/js/client/chaos/test-chaos-load-common.inc

```diff
@@ -330,6 +330,8 @@ function BaseChaosSuite(testOpts) {
         print(shells);
       }
       db._drop(coordination_cn);
+      print(Date() + " tearDown: done.");
```

Comment:
Debug output?

Reply:
yes, without a bit of sequence information in the flow you're completely lost in the test output.